tests/Neo.UnitTests/Network/P2P/Payloads/UT_Header.cs

```diff
@@ -34,6 +34,12 @@ public void Size_Get()
             // blockbase 4 + 64 + 1 + 32 + 4 + 4 + 20 + 4
             // header 1
             Assert.AreEqual(113, uut.Size); // 105 + nonce
+
+            var h = TestUtils.MakeHeader(null, val256);
+            h.Version = (uint)BlockVersion.V1;
+            // blockbase 4 + 64 + 1 + 32 + 4 + 4 + 20 + 4 + 32
+            // header 1
+            Assert.AreEqual(145, uut.Size); // 105 + nonce + prevHash
```

Comment:
still asserts on uut.Size after
  creating the V1 header in h, so the test always compares against the V0 size (113) and fails.

Reply:
Yes, UTs fixes are still in progress.